### PR TITLE
Fix/naming

### LINKED ISSUE #21: Naming incoherency
- Ahead/Behind properties have "Gap" as the total race gap. CarAhead(x) properties have "Gap" as the relative gap on track, and a seperate "RaceGap" property that corresponds to "Gap" from Ahead/Behind properties. This is a bit confusing. 

- The property FuelSaveDelta is used to determine the delta screen pages, which doesnt make much sense

- HotLapPosition with large L. HotlapLivePosition with small L. 

- Actions for pit menu and in car menu controls are labeled "R" and "L" from how I set it up at the SW1 steering wheel. Should change these

Will make a PR to fix these

### PR DESCRIPTION
**Pull request by andreasdahl1987**
Fixes #21

### CODE REVIEW

**Comment by photon1503** [comment]
@andreasdahl1987 There is a compile error now
image

Can you please check?

**Comment by andreasdahl1987** [comment]
Aah, I've missed something I see. It did compile when I tested it a few days ago. Ill have a look

**Review by andreasdahl1987 — COMMENTED** [review]
@photon1503 The errior occured here, I only noticed the NewData/GameData difference. Didnt see the names of the carAhead lists would be changed back to the old version. Will fix!

File: iRacing/iRacing.cs [hunk 1]
```diff
@@ -5091,12 +5091,12 @@ public override void DataUpdate()
 
                 for (int i = 0; i < GameData.OpponentsAheadOnTrack.Count && i < 5; i++)
                 {
-                    carAheadRelative.Add(NewData.OpponentsAheadOnTrack[i].RelativeGapToPlayer);
-                    carAheadGap.Add(NewData.OpponentsAheadOnTrack[i].GaptoPlayer);
-                    carAheadName.Add(NewData.OpponentsAheadOnTrack[i].Name);
-                    carAheadIsInPit.Add(NewData.OpponentsAheadOnTrack[i].IsCarInPit);
-                    carAheadBestLap.Add(NewData.OpponentsAheadOnTrack[i].BestLapTime);
-                    carAheadPosition.Add(NewData.OpponentsAheadOnTrack[i].Position);
+                    carAheadGap.Add(GameData.OpponentsAheadOnTrack[i].RelativeGapToPlayer);
```
Comment (andreasdahl1987) [inline iRacing/iRacing.cs:5094]: @photon1503 I mean here :D

**Comment by andreasdahl1987** [comment]
Try again now

File: iRacing/iRacing.cs (hunk 1 above)
Comment (photon1503) [reply iRacing/iRacing.cs:5094]: great!

**Comment by photon1503** [comment]
> Try again now

everything ok again :-)

**Pull request merged**